src/main/java/frc/robot/commands/drive/AlignWithHub.java

```diff
@@ -44,7 +45,7 @@ public AlignWithHub(DriveSubsystem drive, LimeLightTargeting limelight) {
   public void initialize() {
     // Make sure to reset the heading before resetting the internal PID controller
     m_drive.resetHeading();
-    m_limelight.turnOnLED();
+    m_limelight.setLEDMode(LEDMode.On);
```

Comment:
`TurnOnLImelightLEDs` was the command form of turning on the Limelight LEDs. The reason I named it `setLEDMode` was to reflect that you pass it an enum instead of just a number (I think I've brought that up in the past). Given that we only ever use off and on (not pipeline & blink) the enum might be overkill to use outside of the `Limelight` class, but I'll probably leave them in just in case.